perception/radar_fusion_to_detected_object/src/radar_object_fusion_to_detected_object_node/radar_object_fusion_to_detected_object_node.cpp

```diff
@@ -0,0 +1,218 @@
+  sub_object_ = create_subscription<DetectedObjects>(
+    "~/input/objects", rclcpp::QoS{1},
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onDetectedObjects, this, _1));
+  sub_radar_ = create_subscription<TrackedObjects>(
+    "~/input/radars", rclcpp::QoS{1},
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onRadarObjects, this, _1));
+
+  // Publisher
+  pub_objects_ = create_publisher<DetectedObjects>("~/output/objects", 1);
+
+  // Timer
+  const auto update_period_ns = rclcpp::Rate(node_param_.update_rate_hz).period();
+  timer_ = rclcpp::create_timer(
+    this, get_clock(), update_period_ns,
+    std::bind(&RadarObjectFusionToDetectedObjectNode::onTimer, this));
+}
+
+void RadarObjectFusionToDetectedObjectNode::onDetectedObjects(
+  const DetectedObjects::ConstSharedPtr msg)
+{
+  detected_objects_ = msg;
+}
+void RadarObjectFusionToDetectedObjectNode::onRadarObjects(const TrackedObjects::ConstSharedPtr msg)
+{
+  radar_objects_ = msg;
+}
+
+rcl_interfaces::msg::SetParametersResult RadarObjectFusionToDetectedObjectNode::onSetParam(
+  const std::vector<rclcpp::Parameter> & params)
+{
+  rcl_interfaces::msg::SetParametersResult result;
+
+  try {
+    // Node Parameter
+    {
+      auto & p = node_param_;
+
+      // Update params
+      update_param(params, "node_params.update_rate_hz", p.update_rate_hz);
+    }
+
+    // Core Parameter
+    {
+      // Copy to local variable
+      auto & p = core_param_;
+
+      // Update params
+      update_param(params, "core_params.bounding_box_margin", p.bounding_box_margin);
+      update_param(params, "core_params.split_threshold_velocity", p.split_threshold_velocity);
+      update_param(params, "core_params.velocity_weight_average", p.velocity_weight_average);
+      update_param(params, "core_params.velocity_weight_median", p.velocity_weight_median);
+      update_param(
+        params, "core_params.velocity_weight_target_value_average",
+        p.velocity_weight_target_value_average);
+      update_param(
+        params, "core_params.velocity_weight_target_value_top", p.velocity_weight_target_value_top);
+
+      // Set parameter to instance
+      if (radar_fusion_to_detected_object_) {
+        radar_fusion_to_detected_object_->setParam(core_param_);
+      }
+    }
+  } catch (const rclcpp::exceptions::InvalidParameterTypeException & e) {
+    result.successful = false;
+    result.reason = e.what();
+    return result;
+  }
+
+  result.successful = true;
+  result.reason = "success";
+  return result;
+}
+
+bool RadarObjectFusionToDetectedObjectNode::isDataReady()
+{
+  if (!detected_objects_) {
+    RCLCPP_INFO_THROTTLE(
+      get_logger(), *get_clock(), 1000, "waiting for detected objects data msg...");
+    return false;
+  }
+  if (!radar_objects_) {
+    RCLCPP_INFO_THROTTLE(get_logger(), *get_clock(), 1000, "waiting for radar objects data msg...");
+    return false;
+  }
+
+  if (detected_objects_->header.frame_id != radar_objects_->header.frame_id) {
```

Comment:
Why not transform the objects if necessary?

Reply:
[radar_tracks_msgs_converter](https://github.com/autowarefoundation/autoware.universe/tree/main/perception/radar_tracks_msgs_converter) is used for default radar pipeline and give frame_id as a parameter.
If transform is needed, I'll make new PR.